vpr/src/base/partition_regions.h

```diff
@@ -0,0 +1,15 @@
+#ifndef PARTITION_REGIONS_H
+#define PARTITION_REGIONS_H
+
+/**
+ * Vectors of type PartitionRegions are used to return information on the intersection of regions
```

Comment:
This comment seems odd; need to describe what this stores (a region, which is a union of rectangles).